templates/circuit.yaml

```diff
@@ -309,3 +309,68 @@ spec:
         artifacts:
           - name: circuit-set
             path: /app/circuit_set.json
+
+    - name: create-one-qubit-circuit
+      parent: generic-task
+      inputs:
+        parameters:
+          - name: ry-parameter-1
```

Comment:
We may need to update both templates depending on what we end up doing for the v1 steps to keep everything consistent (please see the suggestions above).

Reply:
I don't think anyone is still using these task templates, but I've left them as they are in case someone needs them for legacy reasons. (They should still work because the the task templates don't import any of the functions modified in this PR.)